watson-developer-cloud/discovery-components
# chore: upgrade carbon components

#### What do these changes do/fix?
Contributes to https://github.ibm.com/Watson-Discovery/disco-issue-tracker/issues/19353

Upgrades version of carbon components consumed by WD react components package, such that a dependency vulnerability is remediated


#### How do you test/verify these changes?
Run example app and make sure nothing broke. Run unit and/or feature tests and make sure none of those broke.
#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
